reinforcement-learning-games/zombsole-gym-example/play-zombsole.nix

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
This was likely copied over from zombsole, and is outdated at this point, and so the file should be removed.